LibSWBF2/Types/List.cpp

```diff
@@ -460,7 +465,11 @@ namespace LibSWBF2
 	template class LIBSWBF2_API Types::List<LVL::terrain::PTCH*>;
 	template class LIBSWBF2_API Types::List<LVL::terrain::VBUF*>;
 
+	template class LIBSWBF2_API Types::List<LVL::light::lght*>;
+    template class LIBSWBF2_API Types::List<LVL::common::DATA*>;
+    template class LIBSWBF2_API Types::List<LVL::common::SCOP*>;
 
+	template class LIBSWBF2_API Types::List<Wrappers::Light>;
 	template class LIBSWBF2_API Types::List<Wrappers::Model>;
 	template class LIBSWBF2_API Types::List<Wrappers::Segment>;
 	template class LIBSWBF2_API Types::List<Wrappers::VertexWeight>;
```

Comment:
There must be a slick way to do this all with a single macro in the source files for each template argument..

Reply:
Maybe there is, but it's not worth looking into that right now IMO. Would be necessary if we'd deal with thousands of generated classes (like in Unreal), but since we're only dealing with less than 50 instances, I think that's a rabbithole.